app/src/androidTest/java/com/conupods/IndoorMaps/ConcreteBuildingTest.java

```diff
@@ -0,0 +1,78 @@
+package com.conupods.IndoorMaps;
+
+import com.conupods.IndoorMaps.ConcreteBuildings.CCBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.VLBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.MBBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.HBuilding;
+import com.conupods.OutdoorMaps.Models.Building.Building;
+import com.google.android.gms.maps.model.LatLng;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConcreteBuildingTest {
+
+    @Test
+    public void CCBuilding(){
+
+        Building CC = CCBuilding.getInstance();
+
+        assertEquals("7141 Sherbrooke West",CC.getAddress());
+        assertEquals("CC",CC.getCode());
+        assertEquals("LOY",CC.getConcreteParent());
+        assertEquals("Central Building",CC.getLongName());
+        assertEquals("CC Building",CC.getName());
+        assertEquals(new ArrayList<>(),CC.getClassRooms());
+        assertEquals(new LatLng(45.458204,-73.6403),CC.getLatLng());
+        assertEquals(new LatLng(45.45863873466155,-73.64075660705566),CC.getOverlayLatLng());
+    }
+
+    @Test
+    public void VLBuilding(){
+
+        Building VL = VLBuilding.getInstance();
+
+        assertEquals("7141 Sherbrooke W",VL.getAddress());
+        assertEquals("VL",VL.getCode());
+        assertEquals("LOY",VL.getConcreteParent());
+        assertEquals("Vanier Library Building",VL.getLongName());
+        assertEquals("VL Building",VL.getName());
+        assertEquals(new ArrayList<>(),VL.getClassRooms());
+        assertEquals(new LatLng(45.459026, -73.638606),VL.getLatLng());
+        assertEquals(new LatLng(45.45890400660071, -73.63919287919998),VL.getOverlayLatLng());
+    }
+
+    @Test
+    public void MBBuilding(){
+
+        Building MB = MBBuilding.getInstance();
+
+        assertEquals("1450 Guy Street",MB.getAddress());
+        assertEquals("MB",MB.getCode());
+        assertEquals("SGW",MB.getConcreteParent());
+        assertEquals("John Molson Building",MB.getLongName());
+        assertEquals("MB Building",MB.getName());
+        assertEquals(new ArrayList<String>(Arrays.asList("MS1 340", "MS1 350", "MS1 130", "MS1 265", "MS1 210", "MS1 430", "MS1 275", "MS1 309", "MS1 410", "MS1 134", "MS1 335", "MS1 115", "MS1 345", "MS1 301", "MS1 315", "MS1 424", "MS1 426", "MS1 437","MSS2 285", "MSS2 105", "MSS2 148", "MSS2 401", "MSS2 445", "MSS2 245", "MSS2 146", "MSS2 465", "MSS2 145", "MSS2 420", "MSS2 440", "MSS2 418", "MSS2 437", "MSS2 230", "MSS2 273", "MSS2 315", "MSS2 435", "MSS2 115", "MSS2 455", "MSS2 410", "MSS2 277", "MSS2 255", "MSS2 210", "MSS2 275", "MSS2 330")),MB.getClassRooms());
+        assertEquals(new LatLng(45.495304, -73.579044),MB.getLatLng());
```

Comment:
the coordinates should be declared in variables for readability and documentation purposes